include/fluent-bit/flb_input_chunk.h

```diff
@@ -50,6 +50,10 @@
 #define FLB_INPUT_CHUNK_TYPE_LOG      0
 #define FLB_INPUT_CHUNK_TYPE_METRIC   1
 
+#ifdef FLB_CHUNK_TRACE
+#define FLB_INPUT_CHUNK_HAS_TRACE     1 << 31
```

Comment:
for preprocessor macros that specify that a feature has been included a build type, we use `FLB_HAVE_....`. e.g:

https://github.com/fluent/fluent-bit/blob/master/CMakeLists.txt#L238